### ISSUE TITLE
Weird error on NVM backup

### ISSUE THREAD

**Issue by heylers** [zwave-js/zwave-js-ui#3273]
### Discussed in https://github.com/zwave-js/zwave-js-ui/discussions/3272

<div type='discussions-op-text'>

<sup>Originally posted by **heylers** September  7, 2023</sup>
Hi.  When I do a manual NVM backup, I get this error: "Error while calling backupNVMRaw: Unknown NVM size - cannot backup! (ZW0209)"   It looks like there's no automatic backup anywhere either, although that's enabled.

Store backup works and are in the folder. 

Everything else works great.  Any idea what to look for? </div>

**Comment by robertsLando** [comment]
Please make a [driver log](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs), loglevel `debug` and attach it here as a file (drag & drop into the text field).

/cc @AlCalzone

**Comment by heylers** [comment]
Here it is.  Log state is right after NVM backup attempt.


[zwavejs_current.log.txt](https://github.com/zwave-js/zwave-js-ui/files/12560557/zwavejs_current.log.txt)

**Comment by AlCalzone** [comment]
Which stick is that?

**Comment by heylers** [comment]
Nortek HUSBZB-1

**Comment by AlCalzone** [comment]
Yeah, you can't backup that one. It doesn't tell how large the NVM is, and we cannot detect when the end is reached, because the data just rolls over and starts from the beginning.

**Comment by robertsLando** [comment]
@AlCalzone Should I close this and/or move it to zwave-js repo

**Comment by AlCalzone** [comment]
I'm okay with moving
